corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -107,61 +106,61 @@ hqDefine('accounting/js/pricing_table', [
                     message = _.template(gettext(
                         "<p>All CommCare subscriptions require a 30 day minimum commitment.</p>" +
                         "<p>Continuing ahead will allow you to schedule your current <%- oldPlan %> " +
-                        "Edition Plan subscription to be paused on <%= dateHtml %>.</p>" +
+                        "Edition Plan subscription to be paused on <strong> <%- date %> </strong></p>" +
                         "<p>If you have questions or if you would like to speak to us about your subscription, " +
-                        "please reach out to <%= emailHtml %>.</p>"
+                        "please reach out to <a href='mailto: <%- invoicingContact %>'><%- invoicingContact %></a>.</p>"
                     ))({
-                        dateHtml: newStartDateHtml,
+                        date: newStartDate,
                         newPlan: newPlan,
                         oldPlan: oldPlan,
-                        emailHtml: mailtoHtml,
+                        invoicingContact: invoicingContact,
                     });
                 } else if (self.oIsNextPlanPaused()) {
                     message = _.template(gettext(
                         "<p>All CommCare subscriptions require a 30 day minimum commitment.</p>" +
                         "<p>Your current <%- oldPlan %> Edition Plan subscription is scheduled to be paused " +
-                        "on <%= dateHtml %>.</p>" +
+                        "on <strong> <%- date %> </strong></p>" +
```

Comment:
changing `dateHtml` to `<strong><%- date %></strong>` also introduces duplication into the codebase -- if someone wants to change the bold styling in the future, it would be easy for one of these changes to be accidentally skipped